tools/run_tests.py

```diff
@@ -148,9 +148,7 @@ def run_shakedown(self, test_dirs, requirements_txt=None, pytest_types='sanity')
 #!/bin/bash
 set -e
 echo "VIRTUALENV CREATE/UPDATE: {venv_path}"
-if  [ ! -f {venv_path}/bin/python3 ] ; then
-    virtualenv -p $(which python3) --always-copy {venv_path}
-fi
+virtualenv -p $(which python3) --always-copy {venv_path}
```

Comment:
For now, I kept the idea that actually worked; use different virtualenvs for different frameworks.